block-storage-attacher/tests/e2e/conf/stage.conf

```diff
@@ -38,20 +38,18 @@ sao01_PVG_CRUISER_PRIVATE_VLAN=2274515
 
 # For plugin build
 GOPATH=$GOPATH						# Read from environment
-BLOCK_PLUGIN_HOME=$GOPATH/src/github.ibm.com/alchemy-containers/armada-storage-block-plugin
+BLOCK_PLUGIN_HOME=$GOPATH/src/github.ibm.com/alchemy-containers/ibmcloud-storage-utilities/block-storage-attacher/tests/e2e/
 
 # Latest Storage Plugin images to be used for e2e testing
 IMAGE_REGISTRY=registry.stage1.ng.bluemix.net
 USER_NAMESPACE=contsto2
-PLUGIN_IMAGE=ibmcloud-block-storage-plugin
-DRIVER_IMAGE=ibmcloud-block-storage-driver
-PLUGIN_BUILD=1.7
-DRIVER_BUILD=1.7
+PLUGIN_IMAGE=ibmcloud-block-storage-attacher
+PLUGIN_BUILD=$PLUGIN_BUILD
 
 # Helm chart install
 KUBECONFIG=$KUBECONFIG
 HELM_CHART_REPO=$HELM_CHART_REPO
-HELM_CHART=$GOPATH/src/github.ibm.com/alchemy-containers/armada-storage-block-plugin/deploy/public-helm-charts/stable/ibmcloud-block-storage-plugin
+HELM_CHART=$GOPATH/src/github.ibm.com/alchemy-containers/ibmcloud-storage-utilities/block-storage-attacher/helm/ibmcloud-block-storage-attacher/
```

Comment:
This should be src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher.
Check the above new path and change accordingly.